event_survey/wizard/event_survey_wizard_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record id="event_survey_generate_view" model="ir.ui.view">
+    <field name="name">wizard.event.survey.form</field>
+    <field name="model">wizard.event.survey</field>
+    <field name="arch" type="xml">
+      <form string="Create Survey">
+        <group>
+          <field name="survey_template_id"
+                 string="Template of survey"
+                 context="{'default_is_template': True}"
```

Comment:
I don't think we should set new surveys as templates by default as we are creating them from a template already.

Reply:
this context affect  on survey which you create from wizard as template, and use it as base to create new  survey for event, if we have domain here only template it's logical that survey created from here is_template